spec/javascripts/helpers/jasmine_examples/SpecHelper.js

```diff
@@ -0,0 +1,15 @@
+beforeEach(function () {
```

Comment:
Weirder and weirder. The helper file is loaded when I run `rake spec:javascript` from the command-line. It's also loaded in the browser *if there's a syntax error in the spec file*, but not otherwise.